certified-connectors/BttnV2/apiDefinition.swagger.json

```diff
@@ -0,0 +1,284 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BttnV2",
+    "description": "The magical bttn that automates your business, improves your customer experience, and so much more\u2026",
+    "version": "1.0",
+    "contact": {
+      "name": "Bttn support",
+      "url": "https://bttn.freshdesk.com/support/solutions",
+      "email": "support@bt.tn"
+    }
+  },
+  "host": "powerautomate.bt.tn",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/company/1": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "getCompanies",
+        "description": "Retrieves the companies from BTTN",
+        "operationId": "GetCompanies",
+        "parameters": [],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/api/action/1/powerAutomate/addSubscription": {
+      "x-ms-notification-content": {
+        "description": "Bttn Webhook Response",
+        "schema": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "event_id": {
+                "type": "string",
+                "description": "event_id"
+              },
+              "action_configuration_id": {
+                "type": "string",
+                "description": "action_configuration_id"
```

Comment:
@WesselBruijnes All the description values in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.